chef_master/source/policyfile.rst

```diff
@@ -217,7 +217,7 @@ A ``Policyfile.rb`` file may contain the following settings:
       named_run_list :update_app, "my_app_cookbook::default"
 
 ``include_policy "NAME", *args``
-   **New in ChefDK 2.4** Specify a policyfile lock to be merged with this policy. ChefDK supports pulling this lock from a local file or from Chef Infra Server. When the policyfile lock is included, its run-lists will appear before the current policyfile's run-list. This setting requires that the solved cookbooks appear as-is from the included policyfile lock. If conflicting attributes or cookbooks are provided, an error will be presented. See `RFC097 <https://github.com/chef/chef-rfc/blob/master/rfc097-policyfile-includes.md>`__ for the full specifications of this feature.
+   **New in ChefDK 2.4, updated in 3.10** Specify a policyfile lock to be merged with this policy. ChefDK supports pulling this lock from a local or remote file, from a Chef Infra Server, or from a git repository. When the policyfile lock is included, its run-list will appear before the current policyfile's run-list. This setting requires that the solved cookbooks appear as-is from the included policyfile lock. If conflicting attributes or cookbooks are provided, an error will be presented. See `RFC097 <https://github.com/chef/chef-rfc/blob/master/rfc097-policyfile-includes.md>`__ for the full specifications of this feature.
```

Comment:
@mattray Could you clarify what was updated in 3.10 for this setting?

Reply:
@mattray You can remove the bit about new in DK 2.4. That's an EOL release. Just explain what's new in 3.10

Reply:
3.10 added support for remote included policyfiles. If we're removing the "New in ChefDK 2.4" section, we may as well remove the "updated in 3.10" part because it's a fairly minor addition to the feature.